src/loopfilter.rs

```diff
@@ -1054,10 +1054,13 @@ impl Rav1dLoopFilterDSPContext {
                         return self;
                     }
 
-                    self.loop_filter_sb[0][0] = dav1d_lpf_h_sb_y_16bpc_avx512icl;
                     self.loop_filter_sb[0][1] = dav1d_lpf_v_sb_y_16bpc_avx512icl;
-                    self.loop_filter_sb[1][0] = dav1d_lpf_h_sb_uv_16bpc_avx512icl;
                     self.loop_filter_sb[1][1] = dav1d_lpf_v_sb_uv_16bpc_avx512icl;
+
+                    if !flags.contains(CpuFlags::SLOW_GATHER) {
+                        self.loop_filter_sb[0][0] = dav1d_lpf_h_sb_y_16bpc_avx512icl;
+                        self.loop_filter_sb[1][0] = dav1d_lpf_h_sb_uv_16bpc_avx512icl;
+                    }
```

Comment:
We need to change this for `BPC8`, too, right?

Reply:
Indeed.